src/index.js

```diff
@@ -0,0 +1,135 @@
+
+let temperature = 20;
+let isCelsius = true;
+
+const increaseButton = document.getElementById('increaseTempControl');
+const decreaseButton = document.getElementById('decreaseTempControl');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const tempValue =  document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const skySelect = document.getElementById('skySelect');
+const sky = document.getElementById('sky');
+const currentTempButton = document.getElementById('currentTempButton');
+
+
+const updateTemprature = () => {
+    // const Celsius = Math.round((temperature-32)*5/9);
+    let temp;
+    if (isCelsius){
+      tempValue.textContent = `${temperature}°C`;
+    } else {
+      temp = getTempInFahrenheit(temperature) 
```

Comment:
Since `temp` is only used in the `else` block, we don't need to initialize it before. 

```suggestion
    if (isCelsius){
      tempValue.textContent = `${temperature}°C`;
    } else {
      const temp = getTempInFahrenheit(temperature) 
```